### PR TITLE
Get rid of activesupport dependency

### PR DESCRIPTION
**Pull request by pyromaniac**
Trying to get rid of this dependency, it is completely unnecessary.

### CODE REVIEW

**Comment by pyromaniac** [comment]
@pkuczynski hey, can you please take a look on this PR and merge if possible? Also, I can't really understand what Codacy wants from me.

**Comment by pkuczynski** [comment]
Looks good to me! waiting for the tests to pass :) I turned off Codacy as it was duplicating code climate

**Comment by pkuczynski** [comment]
@rdubya what do you think about this one?

File: lib/config/configuration.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+module Config
```
[the remark this answers is not in the document]
Comment (pkuczynski) [reply lib/config/configuration.rb]: Ignore it...

**Comment by pyromaniac** [comment]
Perfect, not sure why codeclimate doesn't run fully thouh

**Comment by rdubya** [comment]
@pkuczynski looks good to me

**Comment by pyromaniac** [comment]
Guys, can you please merge it? Really would like to see it merged :) Thanks!

**Comment by pkuczynski** [comment]
What's most important is that all test pass! I will merge it then :) Thanks for your work! @pyromaniac maybe you would like to become a contributor?

**Comment by pyromaniac** [comment]
I'd love to but I'm going to be a bad contributor anyway. I don't really have time even for my OSS unfortunately nowadays :(

**Comment by pkuczynski** [comment]
I know the feeling. Have little time myself and what's worse is that I shifted away from Ruby towards NodeJS. Still, occasional contributor is better than no contributor at all. So maybe you would agree to become such one? :) Especially if you stick to Ruby world...

**Comment by pyromaniac** [comment]
Let's give it a try

**Comment by pkuczynski** [comment]
Cool! Thanks :)

**Pull request merged**